kaolin/graphics/SoftRenderer.py

```diff
@@ -262,51 +284,67 @@ def render(self, vertices, faces, textures=None, mode=None, K=None,
 
         # Fill the back faces of each triangle, if needed
         if self.fill_back:
-            faces = torch.cat((faces, faces[:, :, list(reversed(range(
-                faces.shape[-1])))]), dim=1)
-            textures = torch.cat(
-                (textures, textures.permute(0, 1, 4, 3, 2, 5)), dim=1)
+            faces = torch.cat((faces, faces[:, :, list(reversed(range(faces.shape[-1])))]), dim=1)
+            textures = torch.cat((textures, textures), dim=1)
 
         # Lighting (not needed when we are rendering only depth/silhouette
         # images)
         if mode not in ['depth', 'silhouette']:
             textures = self.lighting(vertices, faces, textures)
 
         # Transform vertices to the camera frame
```

Comment:
not sure this comment is necessary given that method name is already saying it all

Reply:
If you disagree please reply.

Reply:
Do you want to keep this comment ? I don't think the line `self.transform_to_camera_frame(vertices)` is difficult to understand.